docs/06-concepts/06-database/01-connection.md

```diff
@@ -39,26 +45,64 @@ development:
 
 ## Development database
 
-A newly created Serverpod project comes with a preconfigured docker instance with Postgresql. All you have to do is run the following command from the root of the `server` package to start the database.
-
-:::info
-
-[Docker](https://www.docker.com/) is required for this to work.
-
-:::
+A newly created Serverpod project has a preconfigured Docker instance with a Postgres database set up. Run the following command from the root of the `server` package to start the database:
 
 ```bash
 $ docker compose up --build --detach
 ```
 
-To remove the database run (this commands preserve all data).
+To stop the database run:
 
 ```bash
-$ docker compose down
+$ docker compose stop
 ```
 
-To remove the database and __delete__ all data add the `-v` flag.
+To remove the database and __delete__ all associated data, run:
 
 ```bash
 $ docker compose down -v
 ```
+
+## Connecting to a custom Postgres instance
+
+Just like you can connect to the Postgres database inside the Docker container, you can connect to any other Postgres instance. There are a few things you need to take into consideration:
+
+- Make sure that your Postgres instance allows incoming traffic.
```

Comment:
Suggestion: This is a big vague, please add some link or instructions on how to resolve this.